src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,28 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { user, comments, title } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <UserInfo user={user} />
+      </div>
+
+      <p className="PostInfo__body">
+        {post.body}
```

Comment:
why not add post.body into the destructurisation as well?

Reply:
> why not add post.body into the destructurisation as well?

good question=)
Fixed, thanks for reviews!=)